src/README.md

```diff
@@ -0,0 +1,48 @@
+Quirk use
+---------
+Quirks are defined by creating an INI style file in the compiled in quirk location (typically `/usr/share/fwupd/quirks.d`).
+
+The quirk is declared by creating a group based upon the `DeviceInstanceId` or `GUID`
+and then mapping out values to keys.
+
+## All plugins
+All fwupd devices support the following quirks:
+
+| Quirk             | Key                                                                   | Value                                                         | Description                                                                                                                                                                                                                                                                                                                                                                       | Supported values                                                            | Default Value | Minimum fwupd version |
+|-------------------|-----------------------------------------------------------------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|---------------|-----------------------|
+| Plugin            | the device ID, e.g. `DeviceInstanceId=USB\VID_0763&PID_2806`          | the plugin name, e.g. `csr`                                   | Sets the plugin to use for a specific hardware device.                                                                                                                                                                                                                                                                                                                            | n/a                                                                         | n/a           | 1.1.0                 |
```

Comment:
I'm not overwhelmed with the table. Would this be easier to consume if each thing was a level 3 header?

Reply:
Well what about just collapsing the two columns not really used? Still intimidating?

Reply:
I think it's just too much data for the table; you have to scroll in both directions:
![screenshot from 2018-11-11 08-42-14](https://user-images.githubusercontent.com/151380/48310855-c69b0e00-e58d-11e8-8546-349f81552cb9.png)